PaigeVannelli/Rancid-Tomatillos
# Fix/styling update

## Pull Request
<br>
### Description
* Is this a feature or a fix?
FEATURE

* Where should the reviewer start?
MovieBanner.js/css, App.css, MovieDetails.css

* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?
These clean up the style of our search bar, and also add a randomized movie to the top of our main page.

* What outside features or research did you use to implement this fix?
n/a
* How should this be tested?
npm start and view the site on your web browser. Click on the movie banner to ensure it takes you to the correct movie. When clicking on the home button a new movie will be displayed.
<br>
***
#### Please review considerations below:
